spark/src/test/scala/org/apache/spark/sql/delta/DeltaParquetFileFormatSuite.scala

```diff
@@ -112,11 +109,7 @@ class DeltaParquetFileFormatSuite extends QueryTest
           // Select some rows that are deleted and some rows not deleted
           // Deleted row `value`: 0, 200, 300, 756, 10352, 19999
           // Not deleted row `value`: 7, 900
-          val (deletedColumnValue, notDeletedColumnValue) = rowIndexFilterType match {
-            case RowIndexFilterType.IF_CONTAINED => (1, 0)
-            case RowIndexFilterType.IF_NOT_CONTAINED => (0, 1)
-            case _ => (-1, -1) // Invalid, expecting the test to fail.
-          }
+          val (deletedColumnValue, notDeletedColumnValue) = (1, 0)
```

Comment:
You reduced the coverage because it's partially covered by CDCReader tests, isn't it?
Anyway have more tests are always better. I would prefer to keep it.

Reply:
@xupefei Hmm, but right now I don't have control over the rowIndexFilterType though.

Which case does the CDCReader not cover?

Reply:
Ahha I see. We no longer broadcast DVs so there's no IF_CONTAINED or IF_NOT_CONTAINED at this level.